### ISSUE TITLE
Test MVP 6 for accessibility conformance

### ISSUE THREAD

**Issue by awead** [psu-libraries/cho#310]
* run WAVE accessibility toolkit
* keyboard navigation
* voiceover

Child of #165

**Comment by awead** [comment]
TODO
* investigate and implement Siteimprove crawling
* document steps required to perform all the tasks in the wiki

**Comment by cam156** [comment]
Home page is good to go via wave

**Comment by mtribone** [comment]
We have some accessibility issues with the JS for the "add another x" links/buttons when creating collections or resources. Tabbing through the interface jumps over the "add another x" and into the next input. We are also missing some hover/active states to indicate that it is a clickable thing. https://github.com/psu-libraries/cho/issues/957 and
